lib/Swagger/src/Discussion/Api/SitesApi.php

```diff
@@ -334,4 +304,215 @@ public function getSiteWithHttpInfo($site_id)
+                    $e->setResponseObject($data);
+                    break;
+                case 403:
+                    $data = $this->apiClient->getSerializer()->deserialize($e->getResponseBody(), '\Swagger\Client\Discussion\Models\HalProblem', $e->getResponseHeaders());
+                    $e->setResponseObject($data);
+                    break;
+            }
+
+            throw $e;
+        }
+    }
+
+    /**
+     * Operation softDeleteSite
+     *
+     * Soft delete
+     *
+     * @param int $site_id The id of the site (required)
+     * @param string $x_wikia_schwartz  (optional)
+     * @return \Swagger\Client\Discussion\Models\SiteHalResponse
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function softDeleteSite($site_id, $x_wikia_schwartz = null)
+    {
+        list($response) = $this->softDeleteSiteWithHttpInfo($site_id, $x_wikia_schwartz);
+        return $response;
+    }
+
+    /**
+     * Operation softDeleteSiteWithHttpInfo
+     *
+     * Soft delete
+     *
+     * @param int $site_id The id of the site (required)
+     * @param string $x_wikia_schwartz  (optional)
+     * @return Array of \Swagger\Client\Discussion\Models\SiteHalResponse, HTTP status code, HTTP response headers (array of strings)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function softDeleteSiteWithHttpInfo($site_id, $x_wikia_schwartz = null)
+    {
+        // verify the required parameter 'site_id' is set
+        if ($site_id === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $site_id when calling softDeleteSite');
+        }
+        // parse inputs
+        $resourcePath = "/sites/{siteId}/soft-delete";
+        $httpBody = '';
+        $queryParams = array();
+        $headerParams = array();
+        $formParams = array();
+        $_header_accept = $this->apiClient->selectHeaderAccept(array('application/hal+json'));
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(array('application/json'));
+
+        // header params
+        if ($x_wikia_schwartz !== null) {
+            $headerParams['X-Wikia-Schwartz'] = $this->apiClient->getSerializer()->toHeaderValue($x_wikia_schwartz);
+        }
+        // path params
+        if ($site_id !== null) {
+            $resourcePath = str_replace(
+                "{" . "siteId" . "}",
+                $this->apiClient->getSerializer()->toPathValue($site_id),
+                $resourcePath
+            );
+        }
+        // default format to json
+        $resourcePath = str_replace("{format}", "json", $resourcePath);
+
+        
+        // for model (json/xml)
+        if (isset($_tempBody)) {
+            $httpBody = $_tempBody; // $_tempBody is the method argument, if present
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)